config/prod.webpack.config.js

```diff
@@ -1,20 +1,24 @@
 const { resolve } = require('path');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 const config = require('@redhat-cloud-services/frontend-components-config');
 const commonPlugins = require('./plugins');
-const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 
 const { config: webpackConfig, plugins } = config({
   rootFolder: resolve(__dirname, '../'),
-  ...(process.env.BETA && { deployment: 'beta/apps' }),
+  sassPrefix: '.hacCore, .hacInfra',
```

Comment:
@karelhala, what does `sassPrefix` do?  I am unable to find any documentation about it.  

Based on other examples I've seen elsewhere, we probably want:

```suggestion
  sassPrefix: '.hacInfra, .hac-infra',
```
here?